src/map.cpp

```diff
@@ -681,9 +681,9 @@ vehicle *map::move_vehicle( vehicle &veh, const tripoint &dp, const tileray &fac
             veh.tow_data.get_towed()->invalidate_towing( true );
         }
     }
-    // Redraw scene
-    // But only if the vehicle was seen before or after the move
-    if( seen || sees_veh( player_character, veh, true ) ) {
+    // Redraw scene, but only if the player is not engaged in an activity and
+    // the vehicle was seen before or after the move.
+    if( !player_character.activity && ( seen || sees_veh( player_character, veh, true ) ) ) {
```

Comment:
Might need to check it's the auto-drive activity to avoid skipping the animation during other activities. (For example, when player starts eating when driving, or another creatures moves a vehicle while the player is engaged in activity).

Reply:
That was on purpose. I think vehicle animations should be skipped during any activity.
If (in the future) you've got NPCs or AIs driving vehicles around while you're reading you wouldn't want that to slow you down.

Reply:
To put it another way, I think vehicle motion should be treated the same way as creature motion for the purposes of rendering frames during a long-running activity (i.e. not animated).

Reply:
That's a good point.